### DISCUSSION TITLE
Enter entries for past days

### DISCUSSION THREAD

**Discussion by surfer-09** [usememos/memos#1525] [Q&A] [answered]
Hello,

how can i enter text for past days? I can't find an option for this.

Thanks!
surfer ;-)

**Comment by TheNomad11** [comment]
Have you found a way to do so? Here it is said to double-click the time of the date https://github.com/usememos/memos/issues/765 and in the FAQ it is said to use ALT + click https://usememos.com/docs/FAQ#how-to-change-datetime-to-a-memo Neither option works for me. I tried Firefox and Vivaldi

**Comment by Zeng1998** [comment, marked as answer]
@surfer-09 @TheNomad11 

https://github.com/usememos/memos/issues/1017#issuecomment-1537354763
